src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -146,6 +147,63 @@ public bool UseFramebuffer
             }
         }
 
+        /// <summary>
+        /// Set/Get SkyboxUrl.
+        /// If SkyboxUrl is set, the cube map image is loaded and skybox is attached on scene.
+        /// Skybox texture is asynchronously loaded. When loading is finished, ResourcesLoaded is emitted.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SkyboxUrl
+        {
+            set
+            {
+                SetSkybox(value);
+            }
+            get
+            {
+                return skyboxUrl;
+            }
+        }
+
+        /// <summary>
+        /// Set/Get Skybox intensity.
+        /// The skybox intensity is multiplied to the color of skybox texture.
+        /// Default value is 1.0f.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float SkyboxIntensity
+        {
+            set
+            {
+                SetSkyboxIntensity(value);
+            }
+            get
+            {
+                return GetSkyboxIntensity();
+            }
+        }
+
+        /// <summary>
+        /// Set/Get angle of orientation of the skybox.
+        /// If orientation is set, the skybox will be rotate by the Radian orientation along YAxis.
+        /// Default value is 0.0f.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Rotation SkyboxOrientation
+        {
+            set
+            {
+                SetSkyboxOrientation(value);
+            }
+            get
+            {
+                return GetSkyboxOrientation();
+            }
+        }
+
```

Comment:
could you resolve the warnings shown below in several places?
=>
**_Call System.IDisposable.Dispose on object created by 'GetSelectedCamera()' before all references to it are out of scope._**

this can be fixed by adding `using` or by calling `xxx.Dispose();` when it is not used.